packages/eslint-config-godaddy-typescript/index.js

```diff
@@ -0,0 +1,33 @@
+module.exports = {
+  extends: [
+    require('./extends')('eslint-config-godaddy')
+  ],
+  parser: '@typescript-eslint/parser',
+  plugins: ['@typescript-eslint'],
+  overrides: [
+    {
+      files: ['*.js', '*.ts', '*.tsx'],
```

Comment:
Should include `.json` files. Also, do we need `*.tsx` for this variation?

Reply:
Probably not. Similarly I can remove the prop types rule below.